examples/gallery/plot/vectors.py

```diff
@@ -0,0 +1,66 @@
+"""
+Cartesian, circular, and geographic vectors
+------
+
+The :meth:`pygmt.Figure.plot` method can plot Cartesian, circular, and geographic vectors.
+The ``style`` parameter controls vector attributes.
+
+"""
+import numpy as np
+import pygmt
+
+# Create a plot with 15x15 cm basemap, Mercator projection (M) over the continental US
+region = [-127, -64, 24, 53]
+fig = pygmt.Figure()
+fig.coast(
+    region=region,
+    projection="M15c",
+    frame="10.0",
+    borders="1",
+    area_thresh="4000",
+    shorelines="0.25p,black",
+)
+
+
+# plot cartesian vectors with different lengths
+x = np.linspace(-115, -115, 12)  # x vector coordinates
+y = np.linspace(33.5, 42.5, 12)  # y vector coordinates
+xvec = np.linspace(1, 5, 12)  # dx vector data
+yvec = np.zeros(np.shape(y))  # dy vector data
+style = "v0.2+e+a40+gred+h0+p1p,red+z0.35"
```

Comment:
Are they any reason to use `+z0.35` here? For Cartesian vectors, the default input columns are:
```
x y direction length
```

`+z0.35` changes the meaning of the input columns and make the example more difficult to understand.

Please use the default input columns if possible. I believe we will cover different input formats in a longer tutorial, not an example.

Reply:
good idea